### ISSUE TITLE
Basic usage how-to in README.md

### ISSUE THREAD

**Issue by justin-calleja** [json-schema-faker/refaker#2]
Hi. Noob question follows.

I am trying to use [json-schema-faker](https://www.npmjs.com/package/json-schema-faker) to generate JSON documents from a JSON schema which references other JSON schemas.

I am failing to load these other schemas. It seems that loading is always happening through HTTP whereas I need to load these other schemas from the file system (the JSON documents are generated for testing).

Is this supported? Can a basic example of doing so be put in the README.md?

Here's a simple example to have something to work with:

./schemas/user.json:

``` json
{
  "type": "object",
  "$schema": "http://json-schema.org/schema#",
  "properties": {
    "address": {
      "type": "object",
      "$ref": "./address.json"
    },
    "name": {
      "type": "string"
    }
  }
}
```

./schemas/address.json:

``` json
{
  "type": "object",
  "$schema": "http://json-schema.org/schema#",
  "properties": {
    "street": {
      "type": "string"
    }
  }
}
```

Given these files, I have tried several variations of `fakeroot` and `directory` values, as well as different `id` values in the schemas themselves (and the other supported `$schema` value), in the following:

./index.js

``` js
var path = require('path');
var refaker = require('refaker');

var schemasDirPath = path.join(__dirname, 'schemas');
var schemaPath = path.join(__dirname, 'schemas', 'user.json');
var schema = require(schemaPath);

refaker({
  schema: schema,
  fakeroot: schemasDirPath,
  directory: schemasDirPath
}, function(err, refs, schemas) {
  if(err) {
    throw err;
  }

  console.log('refs:', refs);
  console.log('schemas:', schemas);
});
```

It always ends with something like: `Error: cannot reach http://json-schema.org/Users/justin/json-schema/user/schemas` or some other variation over 'http' (or some other impossible path).

`/Users/justin/json-schema/user/` is the project root.

Basically, I don't know how to use this. Can someone point me in the right direction please?

Many thanks!

**Comment by pateketrueke** [comment]
> Basically, I don't know how to use this. Can someone point me in the right direction please?

Well, the fakeroot option should be an URL and not a directory... the main purpose of that option is identifying external references and then load them locally through the filesystem.

BTW, I suggest you to use [json-schema-server](https://github.com/json-schema-faker/json-schema-server) (or even better read the source-code) because it is intended to solve the issue you have.

If you still having issues please let me known.

**Comment by justin-calleja** [comment]
@pateketrueke thanks for the tip re the `fakeroot`.

I tried using the one in the README.md `'http://example.com'` and it's working now... I get it that it's not actually downloading the schema from there... so now I know why it's re**faker**. Wow that took a lot longer to sink in that it should.

Anyway, if anyone wants a basic example: https://github.com/justin-calleja/basic-json-schema-faker-example

Thanks again! And I'll be having a look at [json-schema-server](https://github.com/json-schema-faker/json-schema-server) :)
